common/primitives/src/schema.rs

```diff
@@ -35,6 +38,28 @@ pub enum PayloadLocation {
 	Paginated,
 }
 
+/// Support for up to 16 user-enabled features on a collection.
+#[bitflags]
+#[repr(u16)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+#[derive(Copy, Clone, RuntimeDebug, PartialEq, Eq, Encode, Decode, MaxEncodedLen, TypeInfo)]
+pub enum Grant {
```

Comment:
I agree with Joe to Not call these grants since these are directly associated with Schemas and they are more Properties or Attributes of that schema.